### ISSUE TITLE
Missing old blog posts

### ISSUE THREAD

**Issue by dgw** [YOURLS/website#107]
I've found at least one old blog post that wasn't imported AFAICT: YOURLS/awesome links to a Conditional Toolbar plugin at https://blog.yourls.org/2011/03/yourls-plugin-example-conditional-toolbar/ that now returns 404 on the new site. I can't find it using the search box, either.

**Comment by LeoColomb** [comment]
Indeed, only release blog posts have been migrated, as discussed in our private discussions.
We can still add previous all missing previous blog posts if desired.

For the background, I found some of them quite outdated (I mean not aligned anymore with up-to-date YOURLS behavior), thus I've been focused on release-related ones. That was already challenging 😁

Bonus note, for the conditional toolbar, there is a repo now (this is actually an example of what "outdated" posts can be, where the repo should probably be the only source of trust, I guess).
Anyway, I've submitted a PR for this one: https://github.com/YOURLS/awesome/pull/139

**Comment by dgw** [comment]
FYI the repo's website field also points to the now-missing blog post. I was going remove that broken link, but I can't. 😅

Is there a place I can still access the old posts? Maybe I can help go through them to find the non-release announcements that are still worth migrating.
